recipes/gmtl/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.files import export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "gmtl"
```

Comment:
I think we might need to use the author prefix,

The first result on google for "c++ gmtl" is https://github.com/imvu/gmtl it's available under that name in a few package manages from repology too

Reply:
Could you kindly point me to an example or documentation, how to do this?

Reply:
I am sorry I didn't do that https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-on-recipe-name-collisions

Hmm it's not very clear.


ITS `authour-name` usually the repo owner  name

Reply:
Yes, please, rename this package to `psyinf-gmtl`

Reply:
> Yes, please, rename this package to `psyinf-gmtl`

Thanks for the suggestion. Do I need to rename the folder (subfolder in recipes) accordingly?

Reply:
renamed as suggested. General question: should I close review comments upon implementing them?